### PR TITLE
Version bump to 3.9.0-alpha.0

### PR DESCRIPTION
**Pull request by jeff-phillips-18**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "origin-web-catalog",
-  "version": "0.0.68",
+  "version": "0.0.69",
```
Comment (spadgett) [inline bower.json]: We're switching to `v3.9.0-alpha.0` (1, 2, 3...) for releases as in common

https://github.com/openshift/origin-web-common/releases/tag/v3.9.0-alpha.0

Comment (jeff-phillips-18) [reply bower.json]: OK, thanks. I'll update

**Comment by jeff-phillips-18** [comment]
Should I bump the common version while I'm in there?

**Comment by spadgett** [comment]
Sure. I think @benjaminapetersen is about to cut a v3.9.0-alpha.1 for common

File: bower.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
     "patternfly": ">=3.29.3 <4.0.0",
     "jquery": "~3.2.1",
     "lodash": "~4.17.4",
-    "origin-web-common": "~0.0.70",
+    "origin-web-common": "v3.9.0-alpha.1",
```
Comment (spadgett) [inline bower.json]: I wonder if we can still specify a range for prerelease versions. Does something like `>=v3.9.0-alpha.1 <3.10.0` work?

Comment (spadgett) [reply bower.json]: Seems to work for npm at least testing with https://semver.npmjs.com/

**Comment by jeff-phillips-18** [comment]
Updated to use a range.

File: bower.json [hunk 3]
```diff
@@ -40,7 +40,7 @@
     "patternfly": ">=3.29.3 <4.0.0",
     "jquery": "~3.2.1",
     "lodash": "~4.17.4",
-    "origin-web-common": "~0.0.70",
+    "origin-web-common": ">=v3.9.0-alpha.1 <3.10.0",
```
Comment (benjaminapetersen) [inline bower.json]: +1 def prefer the range, thx!

**Pull request merged**